src/location-conflation.ts

```diff
@@ -7,62 +7,62 @@ import prettyStringify from 'json-stringify-pretty-compact';
 import type { Polygon, MultiPolygon } from 'geojson';
 
 // Type definitions
-type Vec2 = [number, number];
-type Vec3 = [number, number, number];
-type Location = Vec2 | Vec3 | string | number;
+export type Vec2 = [number, number];
+export type Vec3 = [number, number, number];
+export type Location = Vec2 | Vec3 | string | number;
 
-interface FeatureProperties {
+export interface FeatureProperties {
   id: string;
   area: number;
   members?: string[];
   [key: string]: unknown;
 }
 
-type GeoJSONGeometry = Polygon | MultiPolygon;
+export type GeoJSONGeometry = Polygon | MultiPolygon;
 
-interface GeoJSONFeature {
+export interface GeoJSONFeature {
   type: 'Feature';
   id: string;
   properties: FeatureProperties;
   geometry: GeoJSONGeometry;
 }
 
-interface FeatureCollection {
+export interface FeatureCollection {
   type: 'FeatureCollection';
   features: GeoJSONFeature[];
 }
 
-interface LocationSet {
+export interface LocationSet {
   include?: Location[];
   exclude?: Location[];
 }
 
-interface ValidatedLocation {
+export interface ValidatedLocation {
   type: 'point' | 'geojson' | 'countrycoder';
   location: Location;
   id: string;
 }
 
-interface ResolvedLocation extends ValidatedLocation {
+export interface ResolvedLocation extends ValidatedLocation {
   feature: GeoJSONFeature;
 }
 
-interface ValidatedLocationSet {
+export interface ValidatedLocationSet {
   type: 'locationset';
   locationSet: LocationSet;
   id: string;
 }
 
-interface ResolvedLocationSet extends ValidatedLocationSet {
+export interface ResolvedLocationSet extends ValidatedLocationSet {
   feature: GeoJSONFeature;
 }
 
-type ClipOperation = 'UNION' | 'DIFFERENCE';
+export type ClipOperation = 'UNION' | 'DIFFERENCE';
 
-type StringifyOptions = Parameters<typeof prettyStringify>[1];
+export type StringifyOptions = Parameters<typeof prettyStringify>[1];
 
 export class LocationConflation {
-  private _cache: Map<string, GeoJSONFeature>;
+  public _cache: Map<string, GeoJSONFeature>;
```

Comment:
this property is used by iD and Rapid. Not sure if it's better to mark it as `public`, or keep it private and add a new helper method instead? (for manually inserting into the cache)

Reply:
Ah yeah this surprised me too when I updated Rapid's `LocationConflation` recently
https://github.com/bhousel/Rapid/commit/869a45329687499eeb8493de97d968d4e7209ba8

I forgot that we were inserting things directly into the cache.  I agree that making it `public` is fine for now, and sometime soon add some helpers/tests/docs for cache management and to make it clearer that it's allowed to be used that way.